operations/reservations.md

```diff
@@ -716,15 +759,14 @@ Updates information about the specified reservations. Note that if any of the fi
 | `ReservationId` | string | required | Unique identifier of the [Reservation](reservations.md#reservation). |
 | `StartUtc` | [String update value](reservations.md#string-update-value) | optional | Reservation start in UTC timezone in ISO 8601 format. \(or `null` if the start time should not be updated). |
 | `EndUtc` | [String update value](reservations.md#string-update-value) | optional | Reservation end in UTC timezone in ISO 8601 format. \(or `null` if the end time should not be updated). |
-| `AdultCount` | [Number update value](reservations.md#number-update-value) | optional | Count of adults the reservation is for. \(or `null` if the adult count should not be updated). |
-| `ChildCount` | [Number update value](reservations.md#number-update-value) | optional | Count of children the reservation is for. \(or `null` if the child count should not be updated). |
+| `PersonCounts` | array of [Person counts update value](#person-counts-update-value) | optional | Number of people per age category the reservation is for. \(or `null` if the person counts should not be updated). |
```

Comment:
Should not we mentio here, as we discussed, that the old values will be replaced by the new value?

Reply:
I was considering it, it was pretty clear to me from how it's written now, but I can mention it explicitly.

Reply:
I would consider it and ask @MikeAdamsMews how to write it :)